src/test/java/com/smartbear/readyapi/client/DataGenDataSourceTest.java

```diff
@@ -0,0 +1,44 @@
+package com.smartbear.readyapi.client;
+
+import com.smartbear.readyapi.client.model.BooleanDataGenerator;
+import com.smartbear.readyapi.client.model.DataGenDataSource;
+import com.smartbear.readyapi.client.model.DataSourceTestStep;
+import com.smartbear.readyapi.client.model.TestStep;
+import com.smartbear.readyapi.client.teststeps.TestStepTypes;
+import com.smartbear.readyapi.client.teststeps.datasource.datagen.BooleanDataGeneratorBuilder;
+import org.junit.Test;
+
+import static com.smartbear.readyapi.client.TestRecipeBuilder.newTestRecipe;
+import static com.smartbear.readyapi.client.teststeps.TestSteps.dataGenDataSource;
+import static com.smartbear.readyapi.client.teststeps.datasource.datagen.DataGenerators.booleanDataGenerator;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class DataGenDataSourceTest {
+
+    @Test
+    public void buildsRecipeWithDataSourceTestStepWithBooleanDataGenDataSource() throws Exception {
+        TestRecipe recipe = newTestRecipe()
+                .addStep(dataGenDataSource()
+                        .setNumberOfRows(34)
+                        .addDataGenerator(
+                                booleanDataGenerator("property1")
```

Comment:
Should we call it 'booleanTypeProperty()' rather, since other data generators are called 'City', 'UkPostCode' 'USZIPCode' - so it will be rather weird to call 'cityProperty()' but 'cityTypeProperty()' at least will look better.